marketplace-elements.styl

```diff
@@ -77,25 +82,43 @@
     }
 }
 
-.mkt-banner-success {
-    background: $action-success;
-
-    .mkt-banner-content {
-        font-size: 15px;
-        font-weight: 500;
-        padding: 0 40px;
-    }
+.mkt-banner-content {
+    font-size: 15px;
+    font-weight: 500;
+    padding: 0 40px;
 
     small {
         font-size: 15px;
         font-weight: 400;
     }
+}
 
-    a {
-        background-color: $dark-green;
+$banner-colors = $action-success $action-positive $action-wait $action-error;
+$banner-names = success positive "wait" error;
+for $i in (0..3) {
+    $banner-color = $banner-colors[$i];
+    $banner-name = $banner-names[$i];
 
-        &:hover {
-            background-color: darken($dark-green, 10%);
+    .mkt-banner[theme={$banner-name}] {
+        background: $banner-color;
+        a {
+            background-color: darken($banner-color, 10%);
```

Comment:
can we use the colors in the style guide over darken/lightens?

they're currently named -hover and -tapped, which i'm rethinking to just be -light/-dark